### ISSUE TITLE
You can get USB VID:PID for free at openmoko project.

### ISSUE THREAD

**Issue by stanson-ch** [cnlohr/espusb#17]
Nice project, looks very promising.

Since you are opensource, you could easily get free VID:PID here: http://wiki.openmoko.org/wiki/USB_Product_IDs#USB_Vendor_and_Product_IDs

I got one VID:PID (1d50:6030) for one of my projects 5 years ago using one email within 1 hour. Looks like they still have free PIDs and still give out them, so just try to get one. They are nice people, e-mail them right now. :)

**Comment by cnlohr** [comment]
Wouldn't this need to be per-project? Like if someone else made a project, they would use it, or are you speaking for the bootloader feature in this project, specifically?

**Comment by stanson-ch** [comment]
Of course, I'm talking about bootloader only, since this is the only constant thing here. :)

**Comment by cnlohr** [comment]
Hmm... you have a valid point.  Will bookmark this thread.

**Comment by brainstorm** [comment]
http://pid.codes/
